rankers/SimpleAggregateRanker/tests/test_aggregate_ranker.py

```diff
@@ -0,0 +1,166 @@
+        ]
+    }
+    query_chunk_meta = {}
+    match_chunk_meta = {}
+    match_idx = []
+    num_query_chunks = len(query_chunk2match_chunk)
+    for query_chunk_id, matches in query_chunk2match_chunk.items():
+        query_chunk_meta[query_chunk_id] = {'length': num_query_chunks}
+        for c in matches:
+            match_chunk_meta[c['id']] = {'length': c['length']}
+            sign = 1 if is_positive_score else -1
+            match_idx.append((
+                c['parent_id'],
+                c['id'],
+                query_chunk_id,
+                c['score'] * sign,
+            ))
+
+    match_idx_numpy = np.array(
+        match_idx,
+        dtype=[
+            (Chunk2DocRanker.COL_MATCH_PARENT_HASH, np.int64),
+            (Chunk2DocRanker.COL_MATCH_HASH, np.int64),
+            (Chunk2DocRanker.COL_DOC_CHUNK_HASH, np.int64),
+            (Chunk2DocRanker.COL_SCORE, np.float64)
+        ]
+    )
+    return match_idx_numpy, query_chunk_meta, match_chunk_meta
+
+
+def assert_document_order(doc_idx):
+    for i in range(0, len(doc_idx) - 1):
+        assert doc_idx[i][1] > doc_idx[i + 1][1]
+
+
+@pytest.mark.parametrize("chunk_scores,aggregate_function,is_reversed_score,doc_ids,doc_scores", [
+    (
+            chunk_scores(),
+            'max',
+            False,
+            [1, 2, 3],
+            [0.5, 0.25, 0.1]
+    ),
+    (
+            chunk_scores(),
+            'max',
+            True,
+            [3, 2, 1],
+            [1 / (1 + 0.1), 1 / (1 + 0.25), 1 / (1 + 0.5)]
+    ),
+    (
+            chunk_scores(),
+            'min',
+            False,
+            [1, 2, 3],
+            [0.2, 0.15, 0.1]
+    ),
+    (
+            chunk_scores(),
+            'min', True,
+            [3, 2, 1],
+            [1 / (1 + 0.1), 1 / (1 + 0.15), 1 / (1 + 0.2)]
+    ),
+    (
+            chunk_scores(),
+            'mean',
+            False,
+            [1, 2, 3],
+            [1.1 / 3, 0.2, 0.1]
+    ),
+    (
+            chunk_scores(),
+            'mean',
+            True,
+            [3, 2, 1],
+            [1 / (1 + 0.1), 1 / (1 + 0.2), 1 / (1 + 1.1 / 3)]
+    ),
+    (
+            chunk_scores(),
+            'median',
+            False,
+            [1, 2, 3],
+            [0.4, 0.2, 0.1]
+    ),
+    (
+            chunk_scores(),
+            'median',
+            True,
+            [3, 2, 1],
+            [1 / (1 + 0.1), 1 / (1 + 0.2), 1 / (1 + 0.4)]
+    ),
+    (
+            chunk_scores(),
+            'sum',
+            False,
+            [1, 2, 3],
+            [1.1, 0.8, 0.1]
+    ),
+    (
+            chunk_scores(),
+            'sum',
+            True,
+            [3, 2, 1],
+            [1 / (1 + 0.1), 1 / (1 + 0.8), 1 / (1 + 1.1)]
+    ),
+    (
+            chunk_scores(),
+            'prod',
+            False,
+            [3, 1, 2],
+            [0.1, 0.2 * 0.4 * 0.5, 0.25 * 0.25 * 0.15 * 0.15]
+    ),
+    (
+            chunk_scores(),
```

Comment:
just fyi, if it does not variate `chunk_scores` does not need to be parametrized and can be simply passed as a fixture.

Reply:
thanks